CMakeLists.txt

```diff
@@ -215,6 +216,8 @@ cprover_default_properties(
     cbmc
     cbmc-lib
     cpp
+    api_bin
+    bmc_api
```

Comment:
`_` does not follow the current naming convention for modules.
Suggestion: `cprover-api`